### ISSUE TITLE
PIN

### ISSUE THREAD

**Issue by sinaghz** [akexorcist/Android-BluetoothSPPLibrary#26]
Hi, 
thanks for this useful lib,
is there anyway to enter pin for connecting automatically?

**Comment by akexorcist** [comment]
No, bluetooth api on android can't insert pin code automatically.

**Comment by akexorcist** [comment]
Bluetooth API can connecting without pin, but I didn't add into my library yet.
